cachito/workers/pkg_managers/general_js.py

```diff
@@ -340,7 +340,7 @@ def _get_js_component_info_from_nexus(
         component_group = component_group[1:]
     else:
         component_name = name
-        component_group = None
+        component_group = nexus.NULL_GROUP
```

Comment:
I hope this was the original intention and we're not breaking any existing workflows :thinking: